src/usr/initservice/istepdispatcher/istepdispatcher.C

```diff
@@ -1953,6 +1954,16 @@ errlHndl_t IStepDispatcher::sendProgressCode(bool i_needsLock)
     Util::writeScratchReg( SPLESS::MBOX_SCRATCH_REG5,
                            l_scratch5.data32 );
 
+#ifdef CONFIG_PORT80_DEBUG
+    // Starting port 80h value for hostboot isteps.  Each step started will
+    // increase the value by one.
+    static uint8_t port80_val = 0x30;
+    static size_t port80_len = sizeof(port80_val);
+    deviceWrite(TARGETING::MASTER_PROCESSOR_CHIP_TARGET_SENTINEL,
```

Comment:
Is this deliberate to not catch the error here? i.e. did you see errors sometimes?